sycl/doc/extensions/experimental/sycl_ext_oneapi_invoke_simd.asciidoc

```diff
@@ -123,10 +123,15 @@ of a `simd<T, N>` type (e.g. using multiple registers rather than an array).
 We expect implementations of this ABI to build on Clang's `ext_vector_type`,
 rather than specifying the register types directly.
 
-The value of `N` for each `simd` argument to a SIMD function must be the same,
-and represents the sub-group size of the calling kernel.  If a function accepts
-`simd` arguments with multiple `N` values, it cannot be invoked from a SPMD
-kernel and the implementation must issue a diagnostic in this case.
+Each of the following must hold true for a SIMD function and its invoke_simd
+call site:
+- the value of `N` of each `simd` formal parameter corresponding to a non-uniform
+SPMD actual argument (if any) must be the same
+- if the return type of the SIMD function is `simd<T, M>` or `simd_mask<T, M>`,
+  then `M` must match `N` above
```

Comment:
```suggestion
If there are non-uniform actual arguments in `invoke_simd` call site, then each of the following must hold true for the call site and the target SIMD function:
- the value of `N` of each `simd` formal parameter corresponding to a non-uniform
SPMD actual argument (if any) must be the same
- if the return type of the SIMD function is `simd<T, M>` or `simd_mask<T, M>`,
  then `M` must match `N` above
```

Reply:
@Pennycook, does the above change resolve your concern?

Reply:
Yes, LGTM.